src/modules/keyboard.coffee

```diff
@@ -30,6 +30,13 @@ class Keyboard
       @hotkeys[which].push(hotkey)
     )
 
+  removeHotkey: (hotkey) ->
+    hotkey = if Keyboard.hotkeys[hotkey] then Keyboard.hotkeys[hotkey] else hotkey
```

Comment:
Can we toUpperCase() a string to support case insensitive removals ex. `removeHotkey('bold');` Many users might not and need not know the hotkeys object has its keys in all caps.